provisioning/containers/tigervnc/fluxbox/politheme

```diff
@@ -0,0 +1,189 @@
+#This work is licensed under a Creative Commons Attribution-Share Alike 3.0 License.
+#(http://creativecommons.org/licenses/by-sa/3.0/de/deed.en)
+
+--------------------
+
+background:                             aspect
+background.color:                       #003576
+background.colorTo:                     #003576
+
+background.pixmap:      ~/.fluxbox/crownexam.png
```

Comment:
Nice picture :). I would just change the filename in ~/.fluxbox/wallpaper.png or something like that